pact/Pact/Core/IR/Eval/CEK.hs

```diff
@@ -1703,13 +1674,16 @@ eval purity benv term = do
 evalWithinCap
   :: forall e b i
   .  ()
-  => Purity
+  => i
+  -> Purity
   -> BuiltinEnv e b i
-  -> CapToken FullyQualifiedName PactValue
+  -> CapToken QualifiedName PactValue
```

Comment:
Why are we moving from FQN to QN, especially as we need fqn (see line 1686).?

Reply:
Great question, it's because this is particularly so the API is easier to use. This is generally called by chainweb to call something like evaluating something within`(some_module.GAS_PAYER args)`. We don't want our caller to have to deal with getting the fully qual name themselves